src/targets/gpu/kernels/include/migraphx/kernels/math.hpp

```diff
@@ -111,6 +111,8 @@ MIGRAPHX_DEVICE_MATH(sinh, ::sinh)
 MIGRAPHX_DEVICE_MATH(sqrt, ::sqrt)
 MIGRAPHX_DEVICE_MATH(tan, ::tan)
 MIGRAPHX_DEVICE_MATH(tanh, ::tanh)
+MIGRAPHX_DEVICE_MATH(mod, ::mod)
+MIGRAPHX_DEVICE_MATH(fmod, ::fmod)
```

Comment:
I assume you need to expose `fmod` and `remainder`, but I am not sure if hip provides `remainder` so we might need provide our own implementation.

Reply:
they do have a  remainderf 

https://rocmdocs.amd.com/en/latest/ROCm_API_References/HIP-MATH.html?highlight=remainder#remainderf

Reply:
Well thats good.